test/integration/controller/core/clusterqueue_controller_test.go

```diff
@@ -296,7 +300,10 @@ var _ = ginkgo.Describe("ClusterQueue controller", func() {
 						Message: "Can admit new workloads",
 					},
 				},
-			}, ignoreConditionTimestamps))
+				PendingWorkloadsStatus: &kueue.ClusterQueuePendingWorkloadsStatus{
```

Comment:
nit: but I would suggest to use `ignorePendingWorkloadsStatus` here, and check that the list of pending workloads is empty after finishing all workloads in the new dedicated scenario.

Reply:
ignorePendingWorkloadsStatus was added